src/main/java/org/fogbowcloud/arrebol/execution/docker/DockerCommandExecutor.java

```diff
@@ -1,38 +1,45 @@
 package org.fogbowcloud.arrebol.execution.docker;
 
-import static java.lang.Thread.sleep;
-
-import java.util.Objects;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.util.EntityUtils;
 import org.apache.log4j.Logger;
 import org.fogbowcloud.arrebol.execution.docker.request.ExecInstanceResult;
-import org.fogbowcloud.arrebol.execution.docker.request.WorkerDockerRequestHelper;
+import org.fogbowcloud.arrebol.execution.docker.request.HttpWrapper;
+import org.fogbowcloud.arrebol.utils.AppUtil;
+import org.json.JSONObject;
 
-public class DockerCommandExecutor {
+import java.io.UnsupportedEncodingException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+import static java.lang.Thread.sleep;
 
-    private static final Logger LOGGER = Logger.getLogger(DockerCommandExecutor.class);
+public class DockerCommandExecutor {
+    private HttpWrapper httpWrapper;
+    private final Logger LOGGER = Logger.getLogger(DockerCommandExecutor.class);
     private static final long poolingPeriodTimeMs = 300;
-    private final WorkerDockerRequestHelper workerDockerRequestHelper;
 
-    public DockerCommandExecutor(WorkerDockerRequestHelper workerDockerRequestHelper) {
-        this.workerDockerRequestHelper = workerDockerRequestHelper;
+    public DockerCommandExecutor() {
+        this.httpWrapper = new HttpWrapper();
```

Comment:
Is it really necessary to do a `new` in this class? Why not make your methods static?

Reply:
Done.